cmd/tk/workflow.go

```diff
@@ -32,6 +32,7 @@ func applyCmd() *cli.Command {
 	cmd.Flags().BoolVar(&opts.Force, "force", false, "force applying (kubectl apply --force)")
 	cmd.Flags().BoolVar(&opts.Validate, "validate", true, "validation of resources (kubectl --validate=false)")
 	cmd.Flags().BoolVar(&opts.AutoApprove, "dangerous-auto-approve", false, "skip interactive approval. Only for automation!")
+	cmd.Flags().StringVar(&opts.DiffToFile, "diff-to-file", "", "Outputs diff as usual but also outputs it to specified file")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&opts.DiffToFile, "diff-to-file", "", "Outputs diff as usual but also writes it to specified file")
```

Just to avoid repeating `Outputs` twice in one sentence.